highlighters/main/main-highlighter.zsh

```diff
@@ -986,6 +987,21 @@ _zsh_highlight_main_highlighter_highlight_dollar_quote()
   REPLY=$i
 }
 
+# Highlight backtick subshells
+_zsh_highlight_main_highlighter_highlight_backtick()
+{
+  local arg1=$1 i q=\` style
+  i=$arg[(ib:arg1+1:)$q]
```

Comment:
This doesn't account for escaped backticks, e.g., ```LBUFFER='echo `echo \`'``` and ```LBUFFER='echo `echo \``'``` (*sigh* markdown).  The double quote case does handle them; why does the difference?

Reply:
Mostly because double quote already needed to handle `\` for `$` and history expansion. I can add the check.